SCClassLibrary/Platform/Platform.sc

```diff
@@ -91,9 +91,12 @@ Platform {
 		}
 	}
 
-	loadStartupFiles { this.startupFiles.do{|afile|
-		afile = afile.standardizePath;
-		if(File.exists(afile), {afile.load})
+	loadStartupFiles {
```

Comment:
if `project = true`, will it still load startup files relative to the config file?